### PR TITLE
Response body update

### PR DESCRIPTION
**Pull request by marksou**
First draft

### CODE REVIEW

File: tyk-docs/content/transform-traffic/request-body.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ This mechanism operates the same way as the header injection middleware.
 
 ## <a name="request-body-context-data"></a> Context Data
 
-As of version 2.2 Tyk also allows context variables to be injected into the body using the `._tyk_context.` namespace, unlike the context exposed to the URL rewriter and header injector, the body transform can fully iterate through list indices so, for example, calling `_tyk_context.path_parts[0]` in a template will expose the first entry in the `path_parts` list.
+As of version 2.2 Tyk also allows context variables to be injected into the body using the `._tyk_context.` namespace. Unlike the context exposed to the URL rewriter and header injector, the body transform can fully iterate through list indices so, for example, calling `_tyk_context.path_parts[0]` in a template will expose the first entry in the `path_parts` list.
```
Comment (letzya) [inline tyk-docs/content/transform-traffic/request-body.md]: Not sure [0] is the right syntax.

**Review by letzya — CHANGES_REQUESTED** [review]
The context var in body trans is repeated about 3 times out of which two mention iterating through a list (each had different syntax which makes it even more confusing).
(Just search the word “unlike” and see for yourself)
From brief review of the changes I think this page needs a bit of love - consolidate all the related info in one place and not in 3 different places in the same page but might be wrong.

File: tyk-docs/content/transform-traffic/request-body.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/transform-traffic/request-body.md]: Do we just need to remove the brackets?

Comment (asoorm) [reply tyk-docs/content/transform-traffic/request-body.md]: @marksou please try it - I think you need to specify `index _tyk_context.path_parts 0`

Comment (marksou) [reply tyk-docs/content/transform-traffic/request-body.md]: Fixed

**Pull request merged**